src/schema/schema.ts

```diff
@@ -64,6 +64,7 @@ export function createZimbraSchema(
 				accountInfo: client.accountInfo,
 				autoComplete: (_, variables) =>
 					client.autoComplete(variables as AutoCompleteOptions),
+				downloadMessage: (_, variables) => client.downloadMessage(variables),
```

Comment:
looks like smime related changes are showing up here, which shouldn't happen

Reply:
yeah, I was noticing a lot of extra stuff in this PR too

Reply:
Should be resolved now, I needed to merge those commits into my base branch (PREAPPS-824).